src/test/TestMain.h

```diff
@@ -26,22 +26,21 @@
 
 #include "FunctionInvoker.h"
 #include "Global.h"
-#include "ILogger.h"
-#include "Logger.h"
+#include "utility/Logger.h"
```

Comment:
```suggestion
#include "../utility/Logger.h"
```

See https://github.com/DrCpp/DrMock/pull/1#discussion_r549805358